client/tests/webdriver/specs/onboardNewUser.spec.js

```diff
@@ -1,11 +1,11 @@
 import { expect } from "chai"
-import OnboardPage from "../pages/onboard.page"
-
 import CreatePerson from "../pages/createNewPerson.page"
+import OnboardPage from "../pages/onboard.page"
 
 describe("Onboard new user login", () => {
   it('Should show onboard welcome"', () => {
-    OnboardPage.openAsOnboardUser()
+    // give unique name to avoid collision when running multiple tests (without resetting DB)
```

Comment:
Why collision now, it used to work without a unique name right?

Reply:
The collision occurs when you register the onboard user as a new user, we didn't do that before